### DISCUSSION TITLE
Buff bar/relics not showing with in-game setting enabled

### DISCUSSION THREAD

**Discussion by pi-cost** [runelite/runelite#17210] [Q&A] [answered]
Hey all,

My in-game setting for buff bar is not showing in Runelite that should have the trailblazer relics on it. I've reinstalled Runelite and made sure everything is enabled. Any ideas?

**Comment by YvesW** [comment, marked as answer]
You currently have to enable the buff bar on their enhanced client, e.g. the steam client. If you then see the relics on there, it will also show on RuneLite. Jagex did not add an option to their java client to enable or disable the buff bar...

**Reply by pi-cost** [reply]
thanks, that worked.
